pkg/execute/action.go

```diff
@@ -0,0 +1,88 @@
+package execute
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/kubeshop/botkube/pkg/config"
+)
+
+func (e *DefaultExecutor) runActionCommand(ctx context.Context, args []string, clusterName string) (string, error) {
+	if len(args) < 2 {
+		return "", errInvalidCommand
+	}
+
+	validSubCmd := false
+	for _, cmdName := range []string{"action", "actions", "act"} {
+		if args[1] == cmdName {
+			validSubCmd = true
+			break
+		}
+	}
+	if !validSubCmd {
+		return fmt.Sprintf("'%s' is not a valid subcommand", args[1]), nil
+	}
```

Comment:
1. Didn't test it yet, but looks like `@Botkube list Actions` wouldn't work
2. Using map will be more efficient and also simpler

You can replace it with:

below imports define such variable:
```go
var actionResourceNames = map[string]struct{} {
	"action": {},
	"act": {},
	"actions": {},
}
```

replace the selected snippet with:

```suggestion
	cmdRes := strings.ToLower(args[1])
	_, ok := actionResourceNames[cmdRes]
	if !ok {
		return fmt.Sprintf("'%s' is not a valid resource", args[1]), nil
	}
```